src/main/scala/com/kinja/amqp/AmqpConfiguration.scala

```diff
@@ -140,22 +146,32 @@ trait AmqpConfiguration {
 			"direct"
 		}
 
+		val deliveryGuarantee: DeliveryGuarantee = if (exchangeConfig.hasPath("deliveryGuarantee")) {
+			if (exchangeConfig.getBoolean("deliveryGuarantee")) {
+				DeliveryGuarantee.AtLeastOnce
+			} else {
+				DeliveryGuarantee.AtMostOnce
+			}
+		} else {
+			DeliveryGuarantee.AtLeastOnce
+		}
```

Comment:
I have the same problem here as with the boolean type: the value in the configuration is still boolean, making it just as hard to get grasp of. Can you instead use `getString` reading `"at-least-one"` and `"at-most-one"` or something like this?